src/world/Chat/CommandTableStorage.cpp

```diff
@@ -364,6 +364,7 @@ void CommandTableStorage::Init()
         { "fly",                'd', &ChatHandler::HandleDebugFly,                  "",                                                         nullptr, 0, 0, 0 },
         { "disablegravity",     'd', &ChatHandler::HandleDebugDisableGravity,       "",                                                         nullptr, 0, 0, 0 },
         { "featherfall",        'd', &ChatHandler::HandleDebugFeatherFall,          "",                                                         nullptr, 0, 0, 0 },
+        { "speed",              'd', &ChatHandler::HandleDebugSpeed,                "",                                                         nullptr, 0, 0, 0 },
```

Comment:
{ "speed",              'd', &ChatHandler::HandleDebugSpeed,                " speed for unit <creature/player>"